logstash-core/src/main/java/org/logstash/ext/JrubyTimestampExtLibrary.java

```diff
@@ -159,28 +159,22 @@ public IRubyObject ruby_to_json(ThreadContext context, IRubyObject[] args)
             return RubyString.newString(context.runtime,  "\"" + this.timestamp.toIso8601() + "\"");
         }
 
-        public static Timestamp newTimestamp(IRubyObject time)
-        {
-            if (time.isNil()) {
-                return new Timestamp();
-            } else if (time instanceof RubyTime) {
-                return new Timestamp(((RubyTime)time).getDateTime());
-            } else if (time instanceof RubyString) {
-                return new Timestamp(time.toString());
-            } else if (time instanceof RubyTimestamp) {
-                return new Timestamp(((RubyTimestamp) time).timestamp);
-            } else {
-               return null;
-            }
-        }
-
-
         @JRubyMethod(name = "coerce", required = 1, meta = true)
-        public static IRubyObject ruby_coerce(ThreadContext context, IRubyObject recv, IRubyObject time)
-        {
+        public static IRubyObject ruby_coerce(ThreadContext context, IRubyObject recv,
+            IRubyObject time) {
```

Comment:
@jakelandis np :) (I think as a general note, picking a Java code style is in the works already but yes, we can discuss that elsewhere :))